TRIM_E: 0
TRIM_A: 0
TRIM_R: 0
TRIM_T: 0.6

COURSE_KP: 0.7329
COURSE_KD: 0
COURSE_KI: 0.07

ROLL_KP: 1.17
ROLL_KD: -0.13
ROLL_KI: 0

PITCH_KP: 0.1
PITCH_KD: -0.017
PITCH_KI: 0
PITCH_FF: 0

AS_PITCH_KP: -0.0713
AS_PITCH_KD: -0.0635
AS_PITCH_KI: 0

AS_THR_KP: 3.2
AS_THR_KD: 0
AS_THR_KI: 1.0

ALT_KP: 0.045
ALT_KD: 0
ALT_KI: 0.01

BETA_KP: -0.1164
BETA_KD: 0
BETA_KI: -0.0037111
